Source/carthage/main.swift

```diff
@@ -20,6 +20,12 @@ guard ensureGitVersion().first()?.value == true else {
 	exit(EXIT_FAILURE)
 }
 
+let carthageVersion = fetchLatestCarthageVersion()
+
+if carthageVersion.shouldUpgrade() == true {
+	print("Please update to the latest Carthage version: \(carthageVersion.remoteVersion). You currently are on \(carthageVersion.localVersion)")
```

Comment:
We should use `carthage.println` here to format and color this text. (See `Formatting.swift`.)